lib/Backend/DecompilePipe.cpp

```diff
@@ -38,12 +38,27 @@ static void reportProblemNames(const model::Binary &Binary) {
     if (Result)
       OutputStream << *Result << '\n';
   };
+  auto OutputString = [&OutputStream](const std::string &Result) {
+    if (not Result.empty())
+      OutputStream << Result << '\n';
+  };
 
   model::CNameBuilder Builder(Binary);
-  for (const model::Function &F : Binary.Functions())
+  for (const model::Function &F : Binary.Functions()) {
     Output(Builder.warning(F));
+
+    auto VariableNameBuilder = Builder.localVariables(F);
+    for (auto &Variable : F.LocalVariables())
+      OutputString(VariableNameBuilder.name(Variable.Location()).Warning);
+
+    auto LabelNameBuilder = Builder.gotoLabels(F);
+    for (auto &Label : F.GotoLabels())
+      OutputString(LabelNameBuilder.name(Label.Location()).Warning);
+  }
+
   for (const model::Segment &S : Binary.Segments())
     Output(Builder.warning(Binary, S));
+
```

Comment:
How does this appear? Can you give me an example?

Reply:
No, I mean, in the output C. Aren't these warning getting down until the decompiled C?